tests/test_dse.py

```diff
@@ -523,10 +524,11 @@ def test_contracted_shape(self, rotate):
                                               'cire-rotate': rotate}))
 
         # Check code generation
+        trees, bns = get_blocked_nests(op1, ({'x0_blk0'}))
         ys, zs = self.get_params(op1, 'y0_blk0_size', 'z_size')
-        arrays = [i for i in FindSymbols().visit(op1._func_table['bf0']) if i.is_Array]
+        arrays = [i for i in FindSymbols().visit(bns['x0_blk0']) if i.is_Array]
```

Comment:
should u use `trees[1][0]` as well here? in master we trigger both searches from the same root (`op1._func_table['bf0']`), while here the first one is from `bns[...]` and the other one from `trees[1][0]`. What is this `trees[1][0]` ?